.github/workflows/ccpp.yml

```
@@ -59,7 +59,7 @@ jobs:
         CC: cl
 
   test-native:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Reverting to fix the CI. 
```suggestion
    runs-on: ubuntu-22.04
```